apollo-backend/src/test/java/io/logz/apollo/helpers/ModelsGenerator.java

```diff
@@ -272,6 +279,7 @@ public static Deployment createDeployment(Service relatedService, Environment re
         testDeployment.setGroupName(groupName);
         testDeployment.setDeploymentMessage("message-" + Common.randomStr(5));
         testDeployment.setLastUpdate(Date.from(LocalDateTime.now(ZoneId.of("UTC")).atZone(ZoneId.of("UTC")).toInstant()));
+        testDeployment.setStatus(Deployment.DeploymentStatus.PENDING);
```

Comment:
That's the default, no? why you are setting this here? please remove

Reply:
No, when generating like this the status is null